packages/clay-shared/src/useTransitionHeight.ts

```diff
@@ -4,10 +4,32 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-import {removeCollapseHeight, setCollapseHeight} from './helpers';
 import {useEffect, useState} from 'react';
 
-export function useTransition(
+/**
+ * Removes the height style that triggers a transition when collapse
+ * @param collapseElementRef
+ */
+function removeCollapseHeight(collapseElementRef: React.RefObject<any>) {
+	if (collapseElementRef && collapseElementRef.current) {
+		collapseElementRef.current.style.removeProperty('height');
+	}
+}
+
+/**
+ * Adds a height style that triggers a transition when collapsed
+ * @param collapseElementRef
+ */
+function setCollapseHeight(collapseElementRef: React.RefObject<any>) {
+	if (collapseElementRef && collapseElementRef.current) {
+		collapseElementRef.current.setAttribute(
+			'style',
+			`height: ${collapseElementRef.current.children[0].clientHeight}px`
+		);
+	}
+}
+
+export function useTransitionHeight(
```

Comment:
I decided to make this explicit to only "height" right now. In the future I can imagine this being expanded to all transitions, but wanted to narrow our scope now and not make it generic until we actually need it for other use cases.

Reply:
Yeah, makes sense. You did well.